### DISCUSSION TITLE
Can't install Jamulus 3.8.0 on Linux - Ubuntu Studio 20.04.3

### DISCUSSION THREAD

**Discussion by davet0661** [jamulussoftware/jamulus#2254] [Technical Support & Help] [answered]
I have Jamulus 3.7.0 installed on Ubuntu Studio

If I enter "sudo apt install jamulus" it tells me the newest version (3.7.0) is installed

I downloaded the Jamulus 3.8.0 .deb file

It won't install using "Software Install"

I installed GDebi package installer - it describes the file and tells me there are no dependencies, but when I click on "Install Package" it just closes

I've tried from the command line a number of ways, and I get a "Reading package lists... Done" message and then an "unsupported file" message

I'm out of options and tired of searching - has anyone had this issue, or does anyone have any ideas where I can look for an answer?

Thanks

davet0661

**Comment by rdica** [comment]
What command line did you try in a number of ways?

Typically you'd type `sudo dpkg -i file.deb`.
Also make sure the deb file you're attempting to install is in fact a deb pkg.

**Comment by davet0661** [comment, marked as answer]
Solved! By a kind redditor on r/linuxaudio. Opening Gdebi with root privileges way the key. "sudo gdebi-gtk" to open (the GUI version because I'm a Linux n00b), then finding the .deb file, opening it, and installed, first try. That's only two days spent on internet searches and blog posts to install one new program - that's pretty good, compared to most of my experience with Linux : )

**Reply by pljones** [reply]
Thanks for posting the answer!
